pkg/verification/sender/amazon_sns_sender.go

```diff
@@ -27,7 +27,7 @@ func NewAmazonSNSSender(cfg AWSSenderConfiguration) NotificationSender {
 	}
 }
 
-func (s *AmazonSNSSender) SendNotification(_ *gin.Context, content, phoneNumber string) error {
+func (s *AmazonSNSSender) SendNotification(_ *gin.Context, content, phoneNumber, _ string) error {
```

Comment:
Maybe add a TODO note to implement it for AWS?

Reply:
added TODO